tests/unit/test_core/test_pacing.py

```diff
@@ -0,0 +1,104 @@
+"""Tests kytos.core.pacing module."""
+
+import asyncio
+import time
+
+import pytest
+
+from kytos.core.pacing import Pacer, PacerWrapper
+
+
+class TestPacer:
```

Comment:
Without `elastic_window` that you mentioned would be removed, this test suite added roughly 12 secs to the overall unit tests exec (it used to exec in 2 secs total). Let's try to keep this up to at most 4 seconds to keep it quick and snap when running them all.